PX4-Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM px4io/px4-dev-ros-melodic:2021-09-08
 
+RUN mkdir /data
```

Comment:
let's skip this and move to the other dockerfile if not neccesary here

Reply:
cehck if we can remove this.